lib/utils/dates.ts

```diff
@@ -380,7 +380,12 @@ export class Dates {
     const id = year + epiphanyOnSunday.toString();
     if (this.#allDatesBeforeEpiphany[id]) return this.#allDatesBeforeEpiphany[id];
     const start = addDays(this.maryMotherOfGod(year), 1);
-    const end = subtractsDays(this.epiphany(year, epiphanyOnSunday), 1);
+    const epiphany = this.epiphany(year, epiphanyOnSunday);
+
+    // If no days between Mary Mother of God Epiphany
```

Comment:
```suggestion
    // If there are no days between Mary, Mother of God and Epiphany
```

Alternatively: `If Epiphany occurs on 2 January`